api/src/main/java/org/openmrs/module/fhir2/api/impl/FhirImmunizationServiceImpl.java

```diff
@@ -92,73 +92,6 @@ public Immunization create(@Nonnull Immunization newImmunization) {
 		return translator.toFhirResource(obs);
 	}
 	
-	@Override
-	public Immunization update(@Nonnull String uuid, @Nonnull Immunization updatedImmunization) {
```

Comment:
removed the `update` aswell as `patch` since on addition of `applyUpdate`() override here, the rest of the stuff is duplication which can be minimized.

i.e
```
	@Override
	protected Immunization applyUpdate(Obs existingObject, Immunization updatedResource) {
		OpenmrsFhirTranslator<Obs, Immunization> translator = getTranslator();
		
		org.openmrs.Obs updatedObject;
		if (translator instanceof OpenmrsFhirUpdatableTranslator) {
			UpdatableOpenmrsTranslator<org.openmrs.Obs, Immunization> updatableOpenmrsTranslator = (OpenmrsFhirUpdatableTranslator<org.openmrs.Obs, Immunization>) translator;
			updatedObject = updatableOpenmrsTranslator.toOpenmrsType(existingObject, updatedResource);
		} else {
			updatedObject = translator.toOpenmrsType(updatedResource);
		}
		
		validateObject(updatedObject);
		
		return translator.toFhirResource(Context.getObsService().saveObs(updatedObject, "Updated via the FHIR2 API"));
	}
```